modules/ROOT/pages/apicat-create-descriptor-file-manually.adoc

```diff
@@ -185,7 +185,7 @@ contact: # <3>
 versionStrategyConditions:
   majorIncrease:
     branches:
-      - master
+      - root
```

Comment:
not sure what is this, I would not change it
can you please confirm this with someone from the docs team?